index.css

```diff
@@ -61,12 +61,99 @@
     color: #5da8ef;
 }
 
+/* section primary-slider */
+
+.primary-slider {
+    position: relative;
+    padding-top: 43px;
+}
+
+.primary-slider__content {
+     /*linear-gradient(to right, rgba(255, 255, 255, 0.2) 0%, rgba(255, 255, 255, 0) 100%);*/
+     position: absolute;
+     transform: translateY(-50%);
+     top: 50%;
+}
+
+.primary-slider__subtitle {
+    opacity: 0.9;
+    color: #5da8ef;
+    font-size: 9px;
+    font-weight: 700;
+    line-height: 66px;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+}
+
+.primary-slider__title {
+    margin: 0;
+    color: #0b0c11;
+    font-family: "Playfair Display";
```

Comment:
I guess this is an obsolete declaration? Since you have a class that does the same.